eskip/eskip.go

```diff
@@ -96,6 +96,21 @@ type Route struct {
 	Backend string
 }
 
+type RoutePredicate func(*Route) bool
+
+type RouteList []*Route
```

Comment:
ooops, just realized, there is a type routeList []*eskip.Route in master, but it is not public. Does it really have to be public?